src/test/kotlin/com/terraformation/backend/documentproducer/db/ManifestImporterTest.kt

```diff
@@ -79,14 +78,9 @@ class ManifestImporterTest : DatabaseTest() {
 
     @BeforeEach
     fun setUp() {
-      val user = AppUser(usersDao.fetchOneById(cannedAdminId)!!)
-      val context = SecurityContextHolder.getContext()
-
-      oldAuthentication = context.authentication
-      context.authentication = PreAuthenticatedAuthenticationToken(user, "N/A", user.authorities)
-
-      every { permissionChecks.canCreateVariableManifest() } returns true
-      every { permissionChecks.invoke(any()) } answers { nothing }
+      every { user.canCreateVariableManifest() } returns true
+      // TODO is this needed?
+      // every { requirePermissions.invoke(any()) } answers { nothing }
```

Comment:
Apparently not needed!

Reply:
This will be addressed in another PR into the feature branch, for now it is captured as a TODO in the main feature branch PR description.